kubeflow/trainer/rhai/transformers.py

```
@@ -0,0 +1,36 @@
+    pip_index_urls: list[str] = field(
+        default_factory=lambda: list(constants.DEFAULT_PIP_INDEX_URLS)
+    )
+    num_nodes: Optional[int] = None
+    resources_per_node: Optional[dict] = None
+    env: Optional[dict[str, str]] = None
+
+    enable_jit_checkpointing: bool = True
+
+
+def get_trainer_crd_from_transformers_trainer(
+    runtime: types.Runtime,
```

Ruff (0.14.4)

Unused function argument: `runtime`

(ARG001)